code/game/machinery/computer/arcade.dm

```diff
@@ -99,10 +99,10 @@
 	var/name_part1
 	var/name_part2
 
-	name_action = pick("Defeat ", "Annihilate ", "Save ", "Strike ", "Stop ", "Destroy ", "Robust ", "Romance ", "Pwn ", "Own ", "Ban ")

```

Comment:
These are not ERP references.

Reply:
Reclarified the intent of the pull request.

Reply:
You didn't remove "robust". The others could pass as non-metajokes, as they're just part of shared gamer parlance that would make sense in the context of playing an arcade machine,  but robust is definitely one. Ban is one too, so "own" and "pwn" are the only ones of the four I think should stay.